build.xml

```diff
@@ -116,7 +116,7 @@
     <property name="templatedir"  value="${javadir}/template"/>
     <property name="tmptarget"    value="${javadir}/tmp"/>
     <property name="lineendtmp"   value="${tmptarget}/lineend"/>
-    <property name="tempdir"      value="${basedir}/tempdir"/>    <!-- actual tmp files -->
+    <property name="tempdir"      value="${basedir}/temp"/>    <!-- actual tmp files -->
```

Comment:
This was predefined, in the ```clean``` target, but otherwise unused. The value has been changed to use a value that is hardcoded into some tests so all test artifacts are in the same location.